src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt

```diff
@@ -77,6 +78,7 @@ class AmplitudeExperimentService : Disposable {
     }
 
     fun isPartOfExperimentalWelcomeWorkflow(): Boolean {
+        pluginSettings().usageAnalyticsEnabled || return false
```

Comment:
wow! didn't know that trick. How did you found it? Can't find it in official docs....

Reply:
I just wrote it as I would be writing it in shell scripting, and surprisingly it worked...